cmd/server/handlers/ping/handler.go

```diff
@@ -0,0 +1,24 @@
+package ping
+
+import (
+	"context"
+	"gmetrics/internal/database"
+	"gmetrics/internal/helpers"
+	"gmetrics/internal/logger"
+	"net/http"
+	"time"
+)
+
+// Handler возвращает состояние коннекта к базе данных
+func Handler(response http.ResponseWriter, request *http.Request) {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
+	// Проверяем соединение с бд
+	err := database.DB.PingContext(ctx)
+	if err != nil {
+		logger.Log.Error(err)
+		helpers.SetHTTPError(response, http.StatusInternalServerError, []byte{})
+		return
+	}
+	helpers.SetHTTPError(response, http.StatusOK, []byte{})
```

Comment:
У меня название этого метода не бьется с тем, что он делает. Наверное стоит дать ему более информативное название?
```suggestion
	helpers.SetHTTPReponse(response, http.StatusOK, []byte{})
```

Reply:
Функция переросла первоначальный смысл, переименовал